tests/test_ddl.py

```diff
@@ -73,6 +75,7 @@ def test_model_ddl_cap(db):
 
 
 def test_execute_ddl(db: DbBase):
+    skip_json(db)
     mod = db.execute_ddl(
```

Comment:
Don't we call this function in cryptvfs w/ JDB?

Reply:
not skipped anymore